.github/CONTRIBUTING.md

```diff
@@ -127,3 +127,21 @@ You can replicate this process locally with the following commands:
 nf-core create -n testpipeline -d "This pipeline is for testing"
 nf-core lint nf-core-testpipeline
 ```
+
+# To get started:
+
+- Open the repo in [Codespaces](https://github.com/{{ name }})/codespaces)
```

Comment:
This file is in the root of the repo, so is for nf-core/tools.

We need to _also_ put this content into the pipeline template `CONTRIBUTING.md` file. Then it can have `{{ name }}`. Note that there was also a typo with an extra `)` character after the template tag.

```suggestion
- Open the repo in [Codespaces](https://github.com/nf-core/tools/codespaces)
```